src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/HubsLinks.java

```diff
@@ -0,0 +1,28 @@
+package com.wikia.webdriver.testcases.globalnavigationtests;
+
+        import com.wikia.webdriver.common.core.Assertion;
+        import com.wikia.webdriver.common.templates.NewTestTemplate;
+        import com.wikia.webdriver.pageobjectsfactory.pageobject.HomePageObject;
+        import com.wikia.webdriver.pageobjectsfactory.pageobject.globalnav.GlobalNavigationPageObject;
+        import org.openqa.selenium.Dimension;
+        import org.testng.annotations.Test;
+
+public class HubsLinks extends NewTestTemplate {
+
+  private Dimension HUBS_IN_DROPDOWN_RESOLUTION = new Dimension(768, 1024);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "HUBS_IN_DROPDOWN_RESOLUTION" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)